src/rocksdb_store.rs

```diff
@@ -36,27 +36,32 @@ pub enum SetError {
 }
 
 impl RocksDBStore {
-    pub(crate) fn new(config: &StoreConfig) -> Self {
+    pub(crate) fn new(config: Option<&StoreConfig>, custom_path: Option<&Path>) -> Self {
```

Comment:
I think it would be better if these two args were just different variants in the same enum.

Reply:
That'a a way better idea, ill implement that